synapseclient/core/version_check.py

```diff
@@ -148,13 +240,32 @@ def _strip_dev_suffix(version):
     return re.sub(r"\.dev\d+", "", version)
 
 
-def _version_tuple(version, levels=2):
+def _version_tuple(version: str, levels: int = 2) -> tuple:
```

Comment:
Return type should be a bit more specific here:

```
from typing import Tuple

.
.
.
.
-> Tuple[x, y, z]
```

Reply:
The way I understand typing and tuples is that `Tuple[str, str]` implies the return type is two strings, and `Tuple[str, str, str]` is three strings. Since the function can return a `Tuple` of any amount of strings, I can't be more specific.